components/remote_settings/src/client.rs

```diff
@@ -287,6 +373,13 @@ impl ApiClient for ViaductApiClient {
         let resp = self.make_request(attachments_base_url.join(attachment_location)?)?;
         Ok(resp.body)
     }
+
+    fn is_prod_server(&self) -> bool {
+        self.endpoints
+            .root_url
+            .as_str()
+            .starts_with("https://firefox.settings.services.mozilla.com")
```

Comment:
Can't we have constants instead of repeating this?

https://github.com/mozilla/application-services/blob/13908171a87a48c2d52c5f5d73f0a5dda55fb4b0/components/remote_settings/src/config.rs#L69